src/shared/meson.build

```diff
@@ -111,6 +111,7 @@ shared_sources = files(
         'loop-util.c',
         'loopback-setup.c',
         'lsm-util.c',
+        'machine-credential.c',
```

Comment:
You can't just simply move the file around. `Credential` is a too generic name for `shared/` namespace. Please rename the struct to `MachineCredential` or so, which is more descriptive and specific, and change the names of functions accordingly.

Reply:
As above, fixed in 5352365db4